### ISSUE TITLE
You cannot use a second scram implanter for no reason.

### ISSUE THREAD

**Issue by IProduceWidgets** [space-wizards/space-station-14#27416]
## Description

**Reproduction**

Try to use second scram implanter, lose 6 TC instead.

**Screenshots**

**Additional context**

**Comment by Plykiya** [comment]
You're limited to one implant per implant.

**Comment by thebadman4662** [comment]
Yeah, implants are limited to 1 for variety of reasons, one of them being not wasting mindshield on mindshielded people.

**Comment by keronshb** [comment]
> You're limited to one implant per implant type.

This was not the intended design for implants outside of a few.

Some implants should be limited per, some shouldn't. The implants that are limited should have descriptions stating that specifically so people do not end up wasting TC or attempting to implant with more than one.

**Comment by Plykiya** [comment]
> > You're limited to one implant per implant type.
> 
> This was not the intended design for implants outside of a few.
> 
> Some implants should be limited per, some shouldn't. The implants that are limited should have descriptions stating that specifically so people do not end up wasting TC or attempting to implant with more than one.

https://github.com/space-wizards/space-station-14/pull/26250

I mean...

**Comment by K-Dynamic** [comment]
Yeah I bought up this issue in the original pr, I found it weird they made _all_ implants like that as opposed to just mindshield https://github.com/space-wizards/space-station-14/pull/26250#issuecomment-2012128407

**Comment by Simyon264** [comment]
> Yeah I bought up this issue in the original pr, I found it weird they made _all_ implants like that as opposed to just mindshield [#26250 (comment)](https://github.com/space-wizards/space-station-14/pull/26250#issuecomment-2012128407)

As per Emo's review, I made all unable to be implanted more than once. Originally, you could have multiple bikehorns etc.

> These could all just be false, tbh. I dont really see a point in allowing multiple.

**Comment by VasilisThePikachu** [comment]
Some implants should probably "disintegrate" after they are used up like emp. Instead of just allowing multiple. But i agree some implants should be able to be injected multiple times
